File: eFrizer/flutter_efrizer/lib/models/credit_card/credit_card_insert_request.dart
class CreditCardInsertRequest{
  final String cardName;
  final String expiryDate;
  final String cardHolderName;
  final String cvvCode;
  final int clientId;

  CreditCardInsertRequest({
    required this.clientId,
    required this.cardName,
    required this.cardHolderName,
    required this.expiryDate,
    required this.cvvCode
  });

  Map<String, dynamic> toJson() => {
        "cardNumber": cardName,
        "expiryDate": expiryDate,
        "cardHolderName": cardHolderName,
        "cvvCode": cvvCode,
        "clientId": clientId,
      };
}

File: eFrizer/eFrizer.Model/CreditCard/CreditCardInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class CreditCardInsertRequest
    {
        public int ClientId { get; set; }
        public string cardNumber { get; set; }
        public string expiryDate { get; set; }
        public string cardHolderName { get; set; }
        public string cvvCode { get; set; }
    }
}
